plugins/builds/getBuildStatuses.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const stringSchema = joi.string().regex(/^[0-9]+$/);
+const jobIdsSchema = joi.alternatives().try(joi.array().items(stringSchema), stringSchema);
```

Comment:
should this field be required?

Reply:
Are you referring to jobIdsSchema as a whole or to having a minimum length for the array?

Reply:
the jobIdsSchema

Reply:
Not following what you mean by required, do you mean something like `const jobIdsSchema = require(...);`? Or is this referring a property in joi that explicitly makes something required?

Reply:
ah the second one
since the query would be kind of pointless without jobIds passed in, right?

Reply:
Added, thanks!